topics/reference_guide/custom_language_support/navbar.md

```diff
@@ -0,0 +1,38 @@
+[//]: # "title: Navigation Bar"
```

Comment:
the syntax for title must be fixed, see https://plugins.jetbrains.com/docs/intellij/sdk-style.html#page-format

Reply:
Oh, this brackets are so easy to miss. Fixed.